### PR TITLE
DOCS-2046 SCALE: Write Hardware Guidelines

### PR DESCRIPTION
**Pull request by tonyriv3**
DOCS-2046 SCALE: Write Hardware Guidelines

### CODE REVIEW

File: content/SCALE/Introduction/SCALEHardwareGuide.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+
+{{< toc >}}
+
+We now have a SCALE hardware guide alongside our [CORE Hardware Guide]({{< relref "COREHardwareGuide.md" >}}).
+
+From repurposed systems to highly custom builds, the fundamental freedom of TrueNAS is the ability to run it on nearly any x86 computer.
+
+Have something to add? Please open a pull request or issue by clicking the buttons in the side panel!
+
+Here’s to building your best NAS ever in 2021 and beyond!
```
Comment (Mrt134) [inline content/SCALE/Introduction/SCALEHardwareGuide.md:14]:
```suggestion
TrueNAS SCALE and CORE have similar hardware recommendations.
From repurposed systems to highly custom builds, the fundamental freedom of TrueNAS is the ability to run it on nearly any x86 computer.

For hardware built specifically to run TrueNAS, see the [product families](https://www.truenas.com/systems-overview/) that iXsystems provides!

This guide iterates on the [CORE hardware guide]({{< relref "COREHardwareGuide.md" >}}) by adjusting a few recommendations for the Linux base of the SCALE software.
Have something to add? You can edit this article directly by clicking the *Edit this page* button at the top of the article.
Here’s to building your best NAS ever in 2021 and beyond!
```

File: content/SCALE/Introduction/SCALEHardwareGuide.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+**The TrueNAS Team**
+
+## Introduction
+
+The TrueNAS community has a rich ecosystem of advice when it comes to the art and science of choosing the ideal hardware for their favorite storage operating system.
+From the [official Hardware Requirements]({{< relref "COREHardwareGuide.md" >}}) to the [Hardware Recommendations Guide](https://www.ixsystems.com/community/resources/hardware-recommendations-guide.12/) maintained by the community, to countless blog posts, users have a comprehensive, if not overwhelming choice of answers to the simple question:
```
Comment (Mrt134) [inline content/SCALE/Introduction/SCALEHardwareGuide.md:21]:
```suggestion
The TrueNAS community has a rich ecosystem of advice when it comes to the art and science of choosing the ideal hardware for their favorite storage operating system.
From these official Hardware Requirements to the [Hardware Recommendations Guide](https://www.truenas.com/community/resources/hardware-recommendations-guide.12/) maintained by the community, to countless blog posts, users have a comprehensive, if not overwhelming choice of answers to the simple question: 
```

File: content/SCALE/Introduction/SCALEHardwareGuide.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+**The TrueNAS Team**
+
+## Introduction
+
+The TrueNAS community has a rich ecosystem of advice when it comes to the art and science of choosing the ideal hardware for their favorite storage operating system.
+From the [official Hardware Requirements]({{< relref "COREHardwareGuide.md" >}}) to the [Hardware Recommendations Guide](https://www.ixsystems.com/community/resources/hardware-recommendations-guide.12/) maintained by the community, to countless blog posts, users have a comprehensive, if not overwhelming choice of answers to the simple question: 
+
+*“What hardware should I buy?”*
+
+The [TrueNAS Mini](https://www.truenas.com/truenas-mini/) and [TrueNAS R-Series](https://www.ixsystems.com/freenas-certified-servers/) lines of purpose-built TrueNAS systems from iXsystems are the official answers to this question, but they also serve to provide templates for users that want to build their own systems or repurpose existing ones.
```
Comment (Mrt134) [inline content/SCALE/Introduction/SCALEHardwareGuide.md:25]:
```suggestion
The [TrueNAS Mini](https://www.truenas.com/truenas-mini/) and [TrueNAS R-Series](https://www.truenas.com/r-series/) lines of purpose-built TrueNAS systems from iXsystems are the official answers to this question, but they also serve to provide templates for users that want to build their own systems or repurpose existing ones.
```

**Review by Mrt134 — COMMENTED** [review]
Couple suggestions/reworks

**Comment by tonyriv3** [comment]
@Mrt134 
I accepted all the suggestions you made.

**Review by Mrt134 — APPROVED** [review]

**Comment by Mrt134** [comment]
Testing is showing gnupg website is dead - approved for merge.

**Pull request merged**